scripts/verify-partial-repo-build.sh

```diff
@@ -1,4 +1,32 @@
 #!/bin/bash
+set -eux
+
+# show CLI help
+function show_help() {
+  set +x
+  echo " "
+  echo Error: $1
+  echo "Usage:"
+  echo "  ./scripts/verify-partial-repo-build.sh     Exit with code 0 if runtime code repo is well decoupled from the other code"
+  echo "Options:"
+  echo "  --no-revert                                Leaves only runtime code on exit"
+  exit 1
+}
+
+# parse CLI args
+NO_REVERT=
+for i in "$@"
+do
+	case $i in
+		--no-revert)
+			NO_REVERT=true
+			shift
+			;;
+		*)
+			show_help "Unknown option: $i"
+			;;
+	esac
+done
 
 # A script to remove everything from bridges repository/subtree, except:
```

Comment:
@svyatonik 
very nit: can you, please, move this script descripion at the very begining of the script?